packages/kit/src/runtime/server/cookie.spec.js

```diff
@@ -146,4 +147,18 @@ test('cookie names are case sensitive', () => {
 	assert.equal(entryA?.value, 'bar');
 });
 
+test('serialized cookie header should be url-encoded', () => {
+	const href = 'https://example.com';
+	const { cookies, get_cookie_header } = cookies_setup({
+		href,
+		headers: {
+			cookie: 'a=f%C3%BC' // a=fü
+		}
+	});
+	// not that one should do this, but we follow the spec...
```

Comment:
I don't understand what this comment is trying to suggest. Why shouldn't one use `ö` in a cookie value? Maybe better to just remove the comment...
```suggestion
```

Reply:
woops sorry, this is a leftover from the previous test case that I copy pasted initially. removing ...